sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -10,7 +10,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 azure-arm: true
 library-name: Compute
 namespace: Azure.ResourceManager.Compute
-require: https://github.com/Azure/azure-rest-api-specs/blob/883abbe08d313739069c0007eb820aa0a0710748/specification/compute/resource-manager/readme.md
+require: C:\repos\swagger-repos\retry\azure-rest-api-specs-pr\specification\compute\resource-manager\readme.md
```

Comment:
This cannot be committed using a local path.

Reply:
yeah I think they would not merge this in the current state, and should change this PR to a draft until it is ready to merge.
They must use local files for now because their spec is in the private spec repo.